plugins/modules/datacenter.py

```diff
@@ -292,19 +319,16 @@ def update_datacenter(module, client):
     changed = False
     response = None
```

Comment:
i think we can get rid of the forward-declarations of `changed` and `response` because they're set directly at lines 330 and 331